frontend/src/common/utils/menuMapping/index.ts

```diff
@@ -0,0 +1,87 @@
+import { MenuItem, MenuType } from 'common/components/Navbar/components/NavbarMenu/types';
+import { STATIC_NEWS_ID_LINK, STATIC_NEWS_ID_LINK_PATTERN } from 'common/constants/links';
+import {
+  ComponentCommonMenuConfig,
+  GetMenuQuery,
+} from 'common/generated/generated-types';
+import { INTERNAL_PAGE_MAPPING } from 'common/utils/menuMapping/constants';
+import { SubMenuGenerated } from 'common/utils/menuMapping/types';
+import { ImageStrapiUrl } from 'common/utils/urls';
+
+export function mappingSubMenuLink(menu: SubMenuGenerated): string {
```

Comment:
utils ชื่อฟังก์ชันใช้เป็น verb

```suggestion
export function mapSubMenuLink(menu: SubMenuGenerated): string {
```